plutus-core/exe/Main.hs

```diff
@@ -125,8 +125,8 @@ untypedPLC :: Parser Language
 untypedPLC = flag UntypedPLC UntypedPLC (long "untyped" <> short 'u' <> help "Use untyped Plutus Core (default)")
 -- ^ NB: default is always UntypedPLC
 
-languageMode :: Parser Language
-languageMode = typedPLC <|> untypedPLC
+languagemode :: Parser Language
+languagemode = typedPLC <|> untypedPLC
```

Comment:
I liked the previous version more, but whatever.

Reply:
I did that because we've got types with names like `LanguageMode`, parsers for them with names like `langaugeMode`, and variables with names like `languagemode`, and there was some inconsistency in how the latter two were capitalised. It might be less confusing to have `languageModeParser` and so on.